docs/guides/distribution/sign-macos.md

```diff
@@ -13,10 +13,13 @@ If you are not utilizing GitHub Actions to perform builds of OSX DMGs, you will
 
 ## Requirements
 
-- Xcode 11 or above.
-- An Apple Developer account enrolled in the [Apple Developer Program].
+- macOS 10.13.6 or later
+- Xcode 10 or later
+- An Apple Developer account enrolled in the [Apple Developer Program]
 
-## tl;dr
+The prerequisites above are also the minimum conditions for notarization. For more details please read the developer article on [notarizing macOS software before distribution][notarizing macOS software before distribution].
+
+## Setting up Environment Variables
```

Comment:
```suggestion
## tl;dr
```

This section is a short version of the guide in case you don't want to read all the details; idk if there's a better title for it